packages/react-core/src/components/Masthead/MastheadBrand.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Masthead/masthead';
+import { css } from '@patternfly/react-styles';
+
+export interface MastheadBrandProps
+  extends React.DetailedHTMLProps<React.HTMLProps<HTMLAnchorElement>, HTMLAnchorElement> {
+  /** Content rendered inside of the masthead brand. */
+  children?: React.ReactNode;
+  /** Additional classes added to the masthead brand. */
+  className?: string;
+  /** Component type of the masthead brand. */
+  component?: React.ElementType<any> | React.ComponentType<any>;
```

Comment:
Will this allow for people to use react-router with a logo?

Reply:
I believe so, I copied the typing used by `Button` that allows `Link`. Replacing the `"Logo"` string with an icon should result in an icon Link. Should I update the example code to show this or leave the string?

Reply:
Not a bad idea - it will avoid a few questions in the future, i'd imagine. I think would expect that the type of 'component' would need to also allow for `React.ReactNode` for an image or icon to be passed. but I might not totally understand the types you chose.

Reply:
@redallen Will that typing also support images/icons? That's how it was defined in `Button` to allow for `Link` but it may be a different case here.

Reply:
`React.ElementType<any>` covers images/icons via `component={() => <WhateverConsumerWants />}`.